internal/pdatagrpc/logs.go

```diff
@@ -0,0 +1,74 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pdatagrpc
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/internal"
+	otlpcollectorlogs "go.opentelemetry.io/collector/internal/data/protogen/collector/logs/v1"
+)
+
+// TODO: Consider to add `LogsRequest` and `LogsResponse`
```

Comment:
Comment not clear. To add where? Add wrappers for the structs?

Reply:
Right now I return `interface{}` and I think to be better future proof I should create a `LogsResponse` empty struct and return that. So if we ever add things in the OTLP response I can deal with that. Similar for request if we add non data (metadata to the request object in the future I can have that covered).

Reply:
OK, makes sense. Would be good to update the TODO to reflect what you wrote here.

Reply:
Done.